plaidml/bridge/keras/backend_test.py

```diff
@@ -1738,7 +1738,7 @@ def testBigRollup(self, b, x):
         return [b.sum(x, axis=1)]
 
     # Resnet sized tests
-    @opTest([[m(1, 224, 224, 3), m(7, 7, 3, 64)]], do_grads=False)
+    @opTest([[m(1, 229, 229, 3), m(7, 7, 3, 64)]], do_grads=False)
```

Comment:
Do we want to keep this change? @tzerrell

Reply:
No, this is for testing.   Still debugging.  Changes are not ready for review.  I was hoping the "WIP" tag would signify that this was not yet ready for prime time.

Reply:
Please also ignore logging changes --- as is this PR creates WAY to many IVLOG1 logs.  Etc.  Will remove "WIP" when it's in a reviewable state.  Pushed it to PR to get @jbruestle eyeballs on it as my calculateMemInfo changes are causing long runtimes for ResNet50.

Reply:
Yep, no problem.